leo_bringup/config/firmware.yaml

```diff
@@ -7,12 +7,21 @@ firmware:
     pid: { p: 0.0, i: 0.005, d: 0.0 }
     pwm_duty_limit: 100.0
 
+  mecanum_wheels: false
+
   diff_drive:
     wheel_radius: 0.0625
     wheel_separation: 0.33
     angular_velocity_multiplier: 1.91
     input_timeout: 500
 
+  mecanum_drive:
+    wheel_radius: 0.0635
+    wheel_separation: 0.405
+    wheel_base: 0.3052
+    angular_velocity_multiplier: 1.0
+    input_timeout: 500
```

Comment:
Change these parameters to the defaults in the new firmware

Reply:
Done.